### PR TITLE
Payments | Rename models

### PR DESCRIPTION
**Pull request by SultanovRuslan86**
Create new models: 
- BalanceServiceMap 
- PaymentService 
- PaymentCommission
Edit models:
- Transaction to ItemPurchase

### CODE REVIEW

File: backend/payments/apps/transactions/models.py [hunk 1]
```diff
@@ -98,7 +98,7 @@ class TransactionType(models.TextChoices):
         COMPLETED = ('CD', 'COMPLETED')
 
     transaction_id = models.ForeignKey(
```
Comment (lsoulworkerl) [inline backend/payments/apps/transactions/models.py:100]: Rename this field to item_purchase_id

File: backend/payments/apps/transactions/models.py [hunk 2]
```diff
@@ -129,7 +129,7 @@ class Invoice(models.Model):
         default=uuid.uuid4,
         editable=False,
     )
-    transactions = models.ManyToManyField(Transaction)
+    item_purchase = models.ManyToManyField(ItemPurchase)
```
Comment (lsoulworkerl) [inline backend/payments/apps/transactions/models.py:132]: Use plural form - item_purchases

**Comment by lsoulworkerl** [comment]
Rename:
- TransactionHistory -> ItemPurchaseHistory
- TransactionTyep -> ItemPurchaseType
- in model ItemPurchase rename developer_account -> developer_id

And change related names where we use 'transaction', we need to use 'item_purchase'.
For example in ItemPurchase "related_name='transactions_account_to'", change it.

After this changes update migrations.

**Comment by SultanovRuslan86** [comment]
Sorry

**Pull request closed without merging**